### PR TITLE
Correct typos in documentation

### PR DESCRIPTION
**Pull request by mukham12**

### CODE REVIEW

File: attic/TypeBindings.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public JavaType resolveType(Type type) {
 
     /*
     /**********************************************************
-    /* Accesors
+    /* Accessors
```
[the remark this answers is not in the document]
Comment (mukham12) [reply attic/TypeBindings.java:114]: Hey there,

I don’t usually go on a typo hunt 🤣, but whenever I stumble upon one while working on a task it becomes clear to me that there must be others lurking around. That's when I fix them all in one swoop, creating a PR on the fly. 

In regards to your suggestion, while I do appreciate the sentiment, given the relatively low frequency of typos in the documentation, delaying the creation of a PR may not be the best course of action. However, I will take note of your advice and keep it in mind for the future, in the event that I come across a number of distinct errors which can be addressed collectively.

Thank you for your time and attention in this matter.

**Review by cowtowncoder — APPROVED** [review]

File: attic/TypeBindings.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (cowtowncoder) [reply attic/TypeBindings.java:114]: While it'd be nice to get bigger batches I think I'm ok merging this one. Thanks!

[the remark this answers is not in the document]
Comment (mukham12) [reply attic/TypeBindings.java:114]: Thanks Tatu.

[the remark this answers is not in the document]
Comment (cowtowncoder) [reply attic/TypeBindings.java:114]: Thank you @mukham12 for this (and other) contribution!

**Pull request merged**